retrofit/src/main/java/retrofit2/OkHttpCall.java

```diff
@@ -240,7 +250,7 @@ Response<T> parseResponse(okhttp3.Response rawResponse) throws IOException {
 
     ExceptionCatchingResponseBody catchingBody = new ExceptionCatchingResponseBody(rawBody);
     try {
-      T body = responseConverter.convert(catchingBody);
+      T body = responseConverter.convert(catchingBody); //Resource Leak: catching body will lose alias and go out of scope.
```

Comment:
Is `body` not a resource alias of `catchingBody`? If so, I would think this is safe: the responsibility for closing `body` lies with the caller of this method, to whom a Response containing `body` is returned.